lib/main_publiccloud.pm

```diff
@@ -24,12 +24,7 @@ sub load_maintenance_publiccloud_tests {
 
     loadtest "publiccloud/download_repos";
     loadtest "publiccloud/prepare_instance", run_args => $args;
-    if (get_var('PUBLIC_CLOUD_CONSOLE_TESTS')) {
-        loadtest("publiccloud/check_registercloudguest", run_args => $args);
-    }
-    else {
-        loadtest("publiccloud/registration", run_args => $args);
-    }
+    loadtest("publiccloud/registration", run_args => $args);
```

Comment:
```suggestion
    loadtest("publiccloud/registration", run_args => $args) unless get_var('PUBLIC_CLOUD_REGISTRATION_TESTS');
```

:thinking:  ?

Reply:
isn't  `check_registercloudguest` expect to start on unregistered system ? :thinking:

Reply:
ah sorry , all this code collapsing confused me . please ignore this message